### PR TITLE
fix: update toast imports

### PR DESCRIPTION
**Pull request by shavidze**
## Problem

Update unmerged`useToast` imports
## Solution

[Provide a detailed explanation of the solution you are proposing, including any relevant code snippets or screenshots.]

## Checklist

- [x] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [x] Changes have been thoroughly tested
- [ ] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [ ] Any dependencies have been added to the project, if necessary

## Reviewers

@Traxmaxx

### CODE REVIEW

File: dashboard/components/feedback-widget/FeedbackWidget.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import Modal from '@components/modal/Modal';
 import Input from '@components/input/Input';
 import settingsService from '@services/settingsService';
 import Button from '@components/button/Button';
-import useToast from '@components/toast/hooks/useToast';
+import { useToast } from '@components/toast/ToastProvider';
 import Toast from '@components/toast/Toast';
```
Comment (Traxmaxx) [inline dashboard/components/feedback-widget/FeedbackWidget.tsx:11]: Can be removed

File: dashboard/components/feedback-widget/FeedbackWidget.tsx [hunk 2]
```diff
@@ -52,7 +52,7 @@ const useFeedbackWidget = (defaultState: boolean = false) => {
     const [isTakingScreenCapture, setIsTakingScreenCapture] = useState(false);
     const [fileAttachement, setFileAttachement] = useState<File | null>(null);
     const [isSendingFeedback, setIsSendingFeedback] = useState(false);
-    const { toast, setToast, dismissToast } = useToast();
+    const { toast, showToast, dismissToast } = useToast();
```
Comment (Traxmaxx) [inline dashboard/components/feedback-widget/FeedbackWidget.tsx:55]: is `toast` being used? If not please remove

**Review by Traxmaxx — CHANGES_REQUESTED** [review]

**Comment by Traxmaxx** [comment]
Hey! Thanks for the fixes and the PR 🙌 . Several people have fixed this issue in their branch as well and I presume first merge gets in. However I left two minor nitpicks : )

**Review by Traxmaxx — APPROVED** [review]
LGTM

**Review by mlabouardy — APPROVED** [review]

**Pull request merged**